examples/covid_report/tests/report_test.bal

```diff
@@ -0,0 +1,56 @@
+import nuvindu/pipe;
+import ballerina/io;
+import ballerina/test;
+
+@test:Config {
+    groups: ["pipe", "covid_report_example"]
+}
+function testPipeConcurrently() returns error? {
+    pipe:Pipe pipe = new (5);
+    worker A {
+        foreach int i in 1..<5 {
+            error? produce = pipe.produce(i, timeout = 5.00111);
+            if produce is error {
+                io:println(produce);
+            }            
+        }
+    }
+
+    @strand {
+        thread: "any"
+    }
+    worker B {
+        stream<int, error?> intStream = pipe.consumeStream(timeout = 10.12323);
+        IntRecord|error? 'record = intStream.next();
+        int i = 1;
+        while 'record is IntRecord {
+            test:assertEquals('record, i);
+            i+=1;
+            'record = intStream.next();
+        }
```

Comment:
This isn't a good test case. Shall we assert the number of messages received?

Reply:
Sure. I'll modify the test case